src/fr/charly/javaSwingTutorial/MyEclipse.java

```diff
@@ -0,0 +1,107 @@
+package fr.charly.javaSwingTutorial;
+
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+import java.awt.GridLayout;
+
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+import javax.swing.JToolBar;
+import javax.swing.JTree;
+import javax.swing.UIManager;
+import javax.swing.plaf.nimbus.NimbusLookAndFeel;
+
+public class MyEclipse extends JFrame {
+
+	private static final long serialVersionUID = 7768498545170471759L;
+	private static final Component westComponent = null;
+
+	public MyEclipse() {
+		super("My first Swing application");
+		this.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+		this.setSize(600, 400);
+		this.setLocationRelativeTo(null);
+
+		JPanel contentPane = (JPanel) this.getContentPane();
+		// contentPane.setLayout(new BorderLayout();
```

Comment:
Why is this code commented? If it is not needed, remove it. If it is needed, then remove the comments.